src/applications/pensions/components/SectionField.jsx

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import { formatSSN } from 'platform/utilities/ui';
 
 export const convertDateFormat = date => {
-  const [year, month, day] = date.split('-');
-  return `${month}/${day}/${year}`;
+  if (date) {
+    const [year, month, day] = date.split('-');
+    return `${month}/${day}/${year}`;
+  }
+  return date;
 };
 
-const formatPhoneNumber = num =>
-  `(${num.substr(0, 3)}) ${num.substr(3, 3)}-${num.substr(6)}`;
+export const formatPhoneNumber = num => {
```

Comment:
Why would we want to return num in this context if it's falsy?

Reply:
I refactored :call_me_hand: